### PR TITLE
Update k8s docker

### PR DESCRIPTION
**Pull request by crcrpar**
Follow-up of #1998.

### CODE REVIEW

**Comment by hvy** [comment]
#1998 just merged. You can update the diff now 👍

File: examples/kubernetes/simple/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,6 @@
-FROM python:3.5-slim-buster
+FROM optuna/optuna:py3.7-dev
 
 WORKDIR /usr/src/
 
-RUN pip install --no-cache-dir optuna scikit-learn psycopg2-binary
```
Comment (hvy) [inline examples/kubernetes/simple/Dockerfile:5]: What was `psycopg2-binary` used for?

**Comment by crcrpar** [comment]
https://pypi.org/project/psycopg2-binary is for PostgreSQL. I thought we could remove that as I could ran the examples locally though we may have to install that.

**Comment by hvy** [comment]
Hm, okay. It's also explicitly installed in our RDB storage tests on CircleCI https://github.com/optuna/optuna/blob/master/.circleci/config.yml#L220. I tried to run the Docker examples locally to verify but couldn't get the study creation pod to run. Did you also verify using the Dockerfile provided along with the shell script? Cc. @toshihikoyanase

**Comment by crcrpar** [comment]
I'm reverting the changes to two Dockerfiles and only updates PyTorch version of mlflow/Dockerfile then I'm struggling.
Let me close this for now as I can't afford to make things great again now. 🙇

**Pull request closed without merging**